lib/ansible/modules/windows/win_pester.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+module: win_pester
+short_description: Run Pester tests on Windows hosts
+version_added: "2.6"
+description:
+     - Run Pester tests on Windows hosts. Test files have to be available on the remote host.
+options:
+  path:
+    description:
+      - Path to a pester test file or a folder where tests can be found. If the path is a folder, the module will consider all ps1 files as Pester tests.
+    required: true
+  version:
+    description:
+      - Minimum version of the pester module that has to be available on the remote host.
+author:
+    - Erwan Quelin (@erwanquelin)
+'''
+
+RETURN = r'''
+pester_version:
+    description: Version of the pester module found on the remote host.
+    returned: always
+    type: string
+    sample: 4.3.1
+output:
+    description: Results of the Pester tests.
+    returned: success
+    type: list
+    sample: False
+'''
+
+EXAMPLES = r'''
+- name: Get facts
+  setup:
+
+- name: Add Pester module
+  action:
+    module_name: "{{ 'win_psmodule' if ansible_powershell_version >= 5 else 'win_chocolatey' }}"
+    name: Pester
+    state: present
+
+- name: run the pester test provided in the path parameter.
+  win_pester:
+    path: C:\Pester
+
+- name: run all test present in the folder provided in the path parameter.
+  win_pester:
+    path: C:\Pester\
```

Comment:
Unclear to me what the difference is between this entry and the previous.

Also make the task names start with Capital letter. More uniform.

Reply:
You're right, there is no difference...